projects/packages/sync/src/class-sender.php

```diff
@@ -645,7 +646,7 @@ public function do_sync_for_queue( $queue ) {
 					$wp_error = array_pop( $processed_item_ids );
 				}
 				// Also checkin any items that were skipped.
-				if ( count( $skipped_items_ids ) > 0 ) {
+				if ( is_countable( $skipped_items_ids ) && count( $skipped_items_ids ) > 0 ) {
```

Comment:
Might be more straightforward to do `!== array()` instead to make the thing happy. With the existing code here it'll always be an array.